charts/jaeger-operator/templates/role.yaml

```diff
@@ -234,6 +233,13 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - networking.k8s.io
+  resources:
+  - ingressclasses
+  verbs:
+  - get
```

Comment:
Can you please explain why `get` is required? It looks like only `list` is necessary...

https://github.com/jaegertracing/jaeger-operator/blob/03de37980e403ffc4956e9a07c92dc61abc314b3/pkg/autodetect/main.go#L198-L223

Reply:
Thanks for noticing, it should be `list`, `watch` to get rid of all errors from log.
Without watch everything seems to start fine but I see `Failed to watch *v1.IngressClass: unknown` in logs.

I did not notice issues without `get`.